docs/sources/send-data/k8s-monitoring-helm/_index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Kubernetes Monitoring Helm
+menuTitle: Kubernetes Monitoring Helm
+weight: 300
+description: Learn how to collect and store logs from your Kubernetes cluster using Loki.
+killercoda:
+  title: Kubernetes Monitoring Helm
+  description: Learn how to collect and store logs from your Kubernetes cluster using Loki.
+  backend:
+    imageid: kubernetes-kubeadm-2nodes
+---
+
+<!-- INTERACTIVE page intro.md START -->
+
+# Kubernetes Monitoring Helm
+
+One of the primary use cases for Loki is to collect and store logs from your [Kubernetes cluster](https://kubernetes.io/docs/concepts/overview/). These logs fall into three categories:
+
+1. [**Pod logs**](https://kubernetes.io/docs/concepts/cluster-administration/logging/#basic-logging-in-kubernetes): Logs generated by containers otherwise known as logs running in your cluster.
+2. [**Kubernetes Events**](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/): Logs generated by the Kubernetes API server.
+3. [**Node logs**](https://kubernetes.io/docs/concepts/cluster-administration/logging/#using-a-node-logging-agent): Logs generated by the nodes in your cluster.
+
+{{< figure max-width="75%" src="/media/docs/loki/loki-k8s-logs.png" caption="Scraping Kubernetes Logs" alt="Scraping Kubernetes Logs" >}}
+
+In this tutorial, we will deploy [Loki](https://grafana.com/docs/loki/latest/get-started/overview/) and the [Kubernetes Monitoring Helm chart](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/) to collect two of these log types: Pod logs and Kubernetes events. We will also deploy [Grafana](https://grafana.com/docs/grafana/latest/) to visualize these logs.
+
+## Things to know
+
+Before you begin, here are some things you should know:
+
+* **Loki**: Loki can run in a single binary mode or as a distributed system. In this tutorial, we will deploy Loki as a single binary, otherwise known as monolithic mode. Loki can be vertically scaled in this mode depending on the amount of logs you are collecting. Grafana Labs recommends running Loki in a distributed/microservice mode for production use cases to monitor high volumes of logs.
```

Comment:
Amount is for things you can't count.  We can count logs.
```suggestion
* **Loki**: Loki can run in a single binary mode or as a distributed system. In this tutorial, we will deploy Loki as a single binary, otherwise known as monolithic mode. Loki can be vertically scaled in this mode depending on the number of logs you are collecting. Grafana Labs recommends running Loki in a distributed/microservice mode for production use cases to monitor high volumes of logs.
```